Adyen/Model/LegalEntityManagement/BankAccountInfo.cs

```diff
@@ -11,152 +11,73 @@
-        /// <param name="checkCode">The check code of the bank account..</param>
         /// <param name="countryCode">The two-character [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code where the bank account is registered. For example, **NL**..</param>
-        /// <param name="currencyCode">The account&#39;s three-character [ISO currency code](https://docs.adyen.com/development-resources/currency-codes). For example, **EUR**..</param>
-        /// <param name="iban">The international bank account number as defined in the [ISO-13616](https://www.iso.org/standard/81090.html) standard..</param>
-        public BankAccountInfo(string accountNumber = default(string), string accountType = default(string), string bankBicSwift = default(string), string bankCity = default(string), string bankCode = default(string), string bankName = default(string), string branchCode = default(string), string checkCode = default(string), string countryCode = default(string), string currencyCode = default(string), string iban = default(string))
+        public BankAccountInfo(BankAccountInfoAccountIdentification accountIdentification = default(BankAccountInfoAccountIdentification), string accountType = default(string), string countryCode = default(string))
         {
-            this.AccountNumber = accountNumber;
+            this.AccountIdentification = accountIdentification;
             this.AccountType = accountType;
-            this.BankBicSwift = bankBicSwift;
-            this.BankCity = bankCity;
-            this.BankCode = bankCode;
-            this.BankName = bankName;
-            this.BranchCode = branchCode;
-            this.CheckCode = checkCode;
             this.CountryCode = countryCode;
-            this.CurrencyCode = currencyCode;
-            this.Iban = iban;
         }
 
         /// <summary>
-        /// The bank account number (without separators).   When this is provided, the &#x60;branchCode&#x60; is also required.
+        /// Gets or Sets AccountIdentification
         /// </summary>
-        /// <value>The bank account number (without separators).   When this is provided, the &#x60;branchCode&#x60; is also required.</value>
-        [DataMember(Name="accountNumber", EmitDefaultValue=false)]
-        public string AccountNumber { get; set; }
+        [DataMember(Name = "accountIdentification", EmitDefaultValue = false)]
+        public BankAccountInfoAccountIdentification AccountIdentification { get; set; }
 
         /// <summary>
         /// The type of bank account.
         /// </summary>
         /// <value>The type of bank account.</value>
-        [DataMember(Name="accountType", EmitDefaultValue=false)]
+        [DataMember(Name = "accountType", EmitDefaultValue = false)]
         [Obsolete]
         public string AccountType { get; set; }
 
-        /// <summary>
```

Comment:
I noticed a bunch of removals in the BankAccountInfo class, please double-check whether this is intended by confirming it with the openapi specs

Reply:
Nice one. It seems they replaced the BankAccountInfo with oneof models specifying particular accountinfo objects. I'll be sure to test this as soon the specs get fixed for LEM!